encrypted-media-respec.html

```diff
@@ -2077,28 +2086,13 @@
             <li><p>Set the <var>session</var>'s <a def-id="expiration"></a> attribute to <var>expiration time</var>.</p></li>
           </ol>
         </section>
-
-        <section id="close-session">
-          <h4>Close Session</h4>
-          <p>The Close Session algorithm closes the session associated with a <a>MediaKeySession</a> object.</p>
-          <p>Closing a session means that the license(s) and key(s) associated with it are no longer available to decrypt <a def-id="media-data"></a>.
-            All <a>MediaKeySession</a> methods will fail for this object.
-          </p>
-          <p>The following steps are run:</p>
-          <ol>
-            <li><p>Let <var>session</var> be the associated <a>MediaKeySession</a> object.</p></li>
-            <li><p>Let <var>cdm</var> be the CDM instance represented by <var>session</var>'s <var>cdm instance</var> value.</p></li>
-            <li>
-              <p>Use <var>cdm</var> to close the <a href="#key-session">session</a> associated with <var>session</var>.</p>
-              <p class="note">Closing the key session results in the destruction of any license(s) and key(s) that have not been explicitly stored.</p>
-            </li>
-            <li><p>Run the <a def-id="session-closed-algorithm"></a> algorithm on the <var>session</var>.</p></li>
-          </ol>
-        </section>
       
         <section id="session-closed">
           <h4>Session Closed</h4>
           <p>The Session Closed algorithm updates the <a>MediaKeySession</a> state after a <a href="#key-session">session</a> has been closed.</p>
+          <p>
```

Comment:
Similar to an earlier comment (https://github.com/w3c/encrypted-media/pull/184/#discussion_r62946959), it seems like it would be useful to at least note that release message(s) are not sent here (at least pending #180). Although we explicitly do not allow for it, an implementation could send one using the busy-loop algorithm being discussed in #181.

Really, this could apply to any place that the session is closed, but this is the algorithm that will _always_ be called. We could word the note to cover all such cases rather than just this algorithm. Perhaps this would be a good second paragraph in the note div below